<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>621</width>
     <height>621</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>业绩ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>教师ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>工分</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>业绩类型</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>发生时间</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="addperforButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>业绩录入</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>单击表项查看</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>业绩具体信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回上一级</string>
   </property>
  </widget>
  <widget class="QComboBox" name="perforTypeComboBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>全部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>论文</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>软著</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>获奖</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>专利</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>导出成excel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>677</x>
     <y>20</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QComboBox" name="moneyComboBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>全部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0-1000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1000-5000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5000-10000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10000以上</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选中ID可查看业绩详情</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>517</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_text">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>20</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>输入教师工号或姓名后，按回车或者查询按钮进行查询</string>
   </property>
  </widget>
  <widget class="QComboBox" name="happenTimeComboBox">
   <property name="geometry">
    <rect>
     <x>617</x>
     <y>60</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>全部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2017-2018</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2018-2019</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2019-2020</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2020-2021</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>自定义时间段</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
